src/file.rs

```diff
@@ -58,6 +64,14 @@ impl PixelFormat {
             PixelFormat::Indexed { .. } => 1,
         }
     }
+    pub(crate) fn expect_indexed(&self) -> &u8 {
+        match self {
+            PixelFormat::Indexed {
+                transparent_color_index,
+            } => transparent_color_index,
+            _ => panic!("Expected an Indexed PixelFormat"),
```

Comment:
Another panic. That should probably be turned into a `Result`.

Reply:
I was just using this internally in the `write_cel` function. I'll remove this and make it clearer that this should be caught by validation.